lib/prometheus/client/data_stores/direct_file_store.rb

```diff
@@ -68,6 +80,7 @@ def validate_metric_settings(metric_settings)
         end
 
         class MetricStore
+          FILENAME_PREFIX = "prometheus_#{ Prometheus::Client::VERSION }"
```

Comment:
I changed this from `metric` to `prometheus` to reduce the chance of deleting unrelated files.. thoughts?